proto/streamlit/proto/DeckGlJsonChart.proto

```diff
@@ -27,4 +27,11 @@ message DeckGlJsonChart {
 
   // the hash of the json so the the frontend doesn't always have to parse the pydeck json object
   string id = 5;
+
+  /**
+  * Represents the user-provided Mapbox token.
+  * Refer to [Mapbox documentation](https://docs.mapbox.com/help/getting-started/access-tokens/) for details.
+  * Defaults to an empty string, signifying the need to fetch Streamlit's Mapbox token as a fallback.
+   */
```

Comment:
nit: could this be changed to use single-line comments for consistency? I don't think there's any place in our proto files aside from the license headers where we're using multi-line comments

Reply:
Ack. Will do that when this PR has the security assessment completed which I still need to get to... Still have to figure out what to do for SiS for threat detection.